packages/api/ecs/async-operation/tests/test-index.js

```diff
@@ -28,25 +27,6 @@ const { updateAsyncOperation } = require('../index');
 const testDbName = `async_operation_model_test_db_${cryptoRandomString({ length: 10 })}`;
 
 test.before(async (t) => {
-  t.context.dynamoTableName = cryptoRandomString({ length: 10 });
-
-  const tableHash = { name: 'id', type: 'S' };
-  await DynamoDb.createAndWaitForDynamoDbTable({
-    TableName: t.context.dynamoTableName,
-    AttributeDefinitions: [{
-      AttributeName: tableHash.name,
-      AttributeType: tableHash.type,
-    }],
-    KeySchema: [{
-      AttributeName: tableHash.name,
-      KeyType: 'HASH',
-    }],
-    ProvisionedThroughput: {
-      ReadCapacityUnits: 5,
-      WriteCapacityUnits: 5,
-    },
-  });
-
   process.env = { ...process.env, ...localStackConnectionEnv, PG_DATABASE: testDbName };
   const { knex, knexAdmin } = await generateLocalTestDb(testDbName, migrationDir);
   t.context.testKnex = knex;
```

Comment:
Delete the `dynamodbDocClient` setup on L37-40 as well.

Reply:
I'm not sure if this comment was added before one of the above commits, or ...?

Reply:
Or I just hadn't pushed up my in-flight changes. 

5004df7061 should cover the rest of it, plus a few others.